src/App.js

```diff
@@ -7,6 +7,8 @@ import Logout from './components/Logout/logout';
 import LandingPage from './components/landingPage/landingPage';
 import NavBar from './components/navigation/navBar';
 import PageLoader from './components/pageLoader/pageLoader';
+import Profile from './components/profile/Profile';
```

Comment:
@DrKimpatrick and @collinewait, Good work, but it would be best if you followed the naming convention for your filenames.